docs/reference/esql/functions/description/byte_length.asciidoc

```diff
@@ -0,0 +1,5 @@
+// This is generated by ESQL's AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
+
+*Description*
+
+Returns the byte length of a string.
```

Comment:
I see Nik suggested the docs perhaps add a comment about why byte_length is not the same as length. And it would be nice to have a test that shows this.

Reply:
See comment above.